File: Java/j1/src/Employee.java
public class Employee implements Comparable {
   private String fullname;
   private double salary;

public String getFullname() {
    return fullname;
}

public void setFullname(String fullname) {
    this.fullname = fullname;
}

public double getSalary() {
    return salary;
}

public void setSalary(double salary) {
    this.salary = salary;
}

Employee(String fullname,double salary){
    this.fullname=fullname;
    this.salary=salary;
}
@Override
public String  toString(){
return fullname +" "+salary ;
}

@Override
public int compareTo(Object other) {
    if(other instanceof Employee){
        Employee emp =(Employee)other;
       // return this.getFullname().compareTo(emp.fullname);
       return (int)Math.round(this.getSalary()-emp.salary);
    }
    throw new UnsupportedOperationException("Unimplemented method 'compareTo'");
}

    

}

File: C#/StackApp/Models.cs/Employee.cs
public class Employee
{
    public Employee(string firstname, string lastname, int salary)
    {
        Firstname = firstname;
        Lastname = lastname;
        Salary = salary;
    }

  public String Firstname { get; set; }="default";
  public String Lastname { get; set; } ="default lastname";

  public Decimal Salary { get; set; }=0;

    public override string ToString()=> Firstname +" "+ Lastname;
 /* public override string ToString(){
    return string.ConCat(Firstname," ",Lastname);
 }*/

}